robottelo/cli/host.py

```diff
@@ -43,6 +44,31 @@ class Host(Base):
 
     command_base = 'host'
 
+    @classmethod
+    def enc_dump(cls, options):
+        """
+        Dump host's ENC YAML.
+
+        Usage::
+
+            hammer host enc-dump [OPTIONS]
+
+        Options::
+
+             --id ID
+             --location LOCATION_NAME                Location name
+             --location-id LOCATION_ID
+             --location-title LOCATION_TITLE         Location title
+             --name NAME                             Host name
+             --organization ORGANIZATION_NAME        Organization name
+             --organization-id ORGANIZATION_ID       Organization ID
+             --organization-title ORGANIZATION_TITLE Organization title
+             -h, --help                              Print help
+        """
+        cls.command_sub = 'enc-dump'
+        return cls.execute(
+            cls._construct_command(options), output_format='csv')
```

Comment:
@jameerpathan111 any way the output format will be yaml even if you specify json csv or base
so this should be :
```python
return cls.execute(
            cls._construct_command(options), output_format='yaml')
```

Reply:
@ldjebran i have changed the output_format to yaml.
i have not changed the assertion because ```Host.enc_dump({'name': hostname}) ```
 returns list.